packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.sh

```diff
@@ -36,37 +39,37 @@ for i in "${oracles[@]}"; do
     agops oracle accept --offerId "$OFFER_ID" >|"$ORACLE_OFFER"
     agoric wallet print --file "$ORACLE_OFFER"
     agops perf satisfaction --from "$i" --executeOffer "$ORACLE_OFFER" --keyring-backend=test
-    echo "${i}_ORACLE=$OFFER_ID" >> "$HOME/.agoric/envs"
+    echo "${i}_ORACLE=$OFFER_ID" >>"$HOME/.agoric/envs"
```

Comment:
double-checking my (lack of) shell-fu: this has no observable runtime impact, right?

Reply:
right. it was my autoformatter